### PR TITLE
Adds Remembrance Day as a holiday because JamieD12 is a fat plum

### PR DESCRIPTION
**Pull request by nesisknesis**
# Document the changes in your pull request

 Adds remembrance day holiday which puts poppy pins in people's hand at roundstart on that day.

JamieD12 is a fat plum because he merged a PR where I had a review and merged a PR with the DO NOT MERGE label. He also gave both of my PRs merge conflicts.

# Changelog

:cl:  Identification
rscadd: Adds Remembrance Day
/:cl:

### CODE REVIEW

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -437,6 +437,18 @@
...
+/datum/holiday/remembrance
+    name = "Remembrance Day"
+    begin_day = 11
+    begin_month = NOVEMBER
+
+/datum/holiday/remembrance/celebrate()
+    SSticker.OnRoundstart(CALLBACK(src, .proc/roundstart_celebrate))
+
+/datum/holiday/remembrance/proc/roundstart_celebrate()
+    for(var/mob/living/carbon/human/H in GLOB.alive_mob_list)
```
Comment (Bibby0110) [inline code/modules/holiday/holidays.dm]:
```suggestion
    for(var/mob/living/carbon/human/H in GLOB.alive_player_list)
```

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

File: code/modules/holiday/holidays.dm (hunk 1 above)
Comment (nesisknesis) [reply code/modules/holiday/holidays.dm]: i see thank you

Comment (adamsong) [reply code/modules/holiday/holidays.dm]: @TheGamerdk You sure that var exists?

Comment (Bibby0110) [reply code/modules/holiday/holidays.dm]: > @TheGamerdk You sure that var exists?

no

File: code/modules/holiday/holidays.dm [hunk 2]
```diff
@@ -437,6 +437,18 @@
...
+/datum/holiday/remembrance
+    name = "Remembrance Day"
+    begin_day = 11
+    begin_month = NOVEMBER
+
+/datum/holiday/remembrance/celebrate()
+    SSticker.OnRoundstart(CALLBACK(src, .proc/roundstart_celebrate))
+
+/datum/holiday/remembrance/proc/roundstart_celebrate()
+    for(var/mob/living/carbon/human/H in GLOB.alive_player_list)
```
Comment (Bibby0110) [inline code/modules/holiday/holidays.dm]:
```suggestion
    for(var/mob/living/carbon/human/H in GLOB.player_list)
```
derp

**Review by Bibby0110 — APPROVED** [review]

**Pull request merged**